src/main/java/com/synopsys/integration/rest/service/IntJsonTransformer.java

```diff
@@ -78,6 +80,9 @@ public IntJsonTransformer(final Gson gson, final IntLogger logger) {
 
     public <C extends IntRestComponent> C getComponentAs(JsonObject jsonObject, Type responseType) throws IntegrationException {
         final String json = gson.toJson(jsonObject);
+        if (null == jsonObject || StringUtils.isBlank(json)) {
```

Comment:
Seems like there might be some risk in passing null to gson.toJson()... do we want the jsonObject null check before passing it?

Reply:
gson.toJson(null) produces "null".  I did it this way to handle the jsonObject as well as the json string being blank. I can remove the json string check and just move the null check on the jsonObject up if that is prefferred.

Reply:
OK cool, sounds like this covers what actually happens, so I'm fine with it.